app/helpers/twitter_helper.rb

```diff
@@ -0,0 +1,46 @@
+module TwitterHelper
+  def twitter_enabled
+    if Rails.application.secrets.twitter
+      Rails.application.secrets.twitter['enabled'] == true
+    end
+  end
+
+  def twitter_client
+    Twitter::REST::Client.new do |config|
+      config.bearer_token = Rails.application.secrets.twitter['bearer_token']
+    end
+  end
+
+  def twitter_handle
+    Rails.application.secrets.twitter['twitter_handle']
+  end
+
+  def twitter_user_info
+      $twitter_user_info ||= []
```

Comment:
Ahh... I see. This is also used for caching - so it definitly is something else than an instance variable with an `@` as that would go away after each request. Will have to read up on it.